src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/email/keystone/KeystoneV3IdentityRequestHelper.java

```diff
@@ -35,7 +36,7 @@
     private static final String ID_PROP = "id";
     private static final String V3_TOKENS_ENDPOINT_PATH = "/v3/auth/tokens";
 
-    public static String createAccessId(Map<String, String> credentials) throws JSONException, IOException {
+    public static IdentityToken createAccessId(Map<String, String> credentials) throws JSONException, IOException {
```

Comment:
This method now seems to do more than just return an access id, change the name to `createIdentityToken` or something.

Reply:
Done.